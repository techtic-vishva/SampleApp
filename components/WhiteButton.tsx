import OrangeButton, { ButtonProps } from './OrangeButton'

export default function (props: ButtonProps) {
	return (
		<OrangeButton
			{...props}
			textColorOverride="white"
			outterStyle={Object.assign({ backgroundColor: 'transparent', borderColor: 'white' }, props.outterStyle || {})}
		/>
	)
}
