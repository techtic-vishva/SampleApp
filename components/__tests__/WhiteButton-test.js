import * as React from 'react'
import renderer from 'react-test-renderer'
import WhiteButton from '../WhiteButton'

it('render when passing the props', () => {
	const tree = renderer
		.create(
			<WhiteButton
				onPress={() => {
					console.log('onPress')
				}}
				title={''}
				icon={''}
				outterStyle={{}}
			/>
		)
		.toJSON()
	expect(tree).toMatchSnapshot()
})
