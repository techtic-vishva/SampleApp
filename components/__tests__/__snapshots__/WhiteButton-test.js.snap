// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render when passing the props 1`] = `
<View
  accessibilityState={
    Object {
      "disabled": false,
    }
  }
  accessible={true}
  collapsable={false}
  focusable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "transparent",
      "borderColor": "white",
      "borderRadius": 10,
      "borderWidth": 1,
      "display": "flex",
      "flexDirection": "row",
      "justifyContent": "center",
      "opacity": 1,
      "paddingHorizontal": 30,
      "paddingVertical": 12,
      "width": "50%",
    }
  }
>
  
  <Text
    style={
      Array [
        Object {
          "color": "#fff",
          "fontFamily": "objektiv",
        },
        Object {
          "color": "white",
          "fontSize": 18,
        },
      ]
    }
  >
    
  </Text>
  
</View>
`;
